### ISSUE TITLE
Enhancement: parameter controling autoupdate in config file

### ISSUE THREAD

**Issue by ebojali** [RainbowMiner/RainbowMiner#53]
Hi.

Please add a config parameter that allows autoupdate as soon as a New update is available. I don't want to login on my rig all the time and press U. A parameter "allowAutoUpdate" could be introducera.

**Comment by Byrd1083** [comment]
Yes me to, when I first ran it, it said miner needs update. It attempted to automate this but failed. I had to manually update with U to get it to work.

**Comment by RainbowMiner** [comment]
This is a good idea. I will implement it at once.

RBM

**Comment by RainbowMiner** [comment]
Implemented:
- set "EnableAutoUpdate" to "1" in config.txt or use configuration->common
